File: HackerRank/py/30days/day3.py
if __name__ == '__main__':
    N = int(input().strip())
    if N % 2 != 0:
        print("Weird")
    elif N >= 2 and N <= 5:
        print("Not Weird")
    elif N <= 20:
        print("Weird")
    else:
        print("Weird")

File: HackerRank/30days/go/day3.go
package main

import "fmt"

func main() {
	number := 10
	if number%2 != 0 {
		fmt.Println("Weird")
	} else if number >= 2 && number <= 5 {
		fmt.Println("Not Weird")
	} else if number >= 6 && number <= 20 {
		fmt.Println("Weird")
	} else if number > 20 {
		fmt.Println("Not Weird")
	}
}
